src/components/Footer/Footer.js

```diff
@@ -66,20 +69,28 @@ const Footer = () => (
 
         <ul className={styles['footer__list']}>
           <li className={styles['footer__list__first__row']}>
-            <a target='blank' className={styles['footer__list__link']} href='#'>
-              <img src='https://www.clipartmax.com/png/middle/35-351662_update-to-our-address-email-icon-png.png'></img>
+            <Link to='/#' className={styles['footer__list__link']}>
```

Comment:
This is not a Link, it's an `a` tag because it features a hyperlink to the email address. Restore the default code and implement `mailto` inside the `a` tag.